src/workspaces/SettingsModal/LogBucketRetention.tsx

```diff
@@ -0,0 +1,67 @@
+import { ExternalLink, useUniqueId } from '@terra-ui-packages/components';
+import React, { ReactNode } from 'react';
+import { NumberInput } from 'src/components/input';
+import { FormLabel } from 'src/libs/forms';
+
+interface LogBucketRetentionProps {
+  retentionPeriodInDays: number | null;
+  setRetentionPeriod: (retentionPeriodInDays: number) => void;
+  isOwner: boolean;
+}
+
+const LogBucketRetention = (props: LogBucketRetentionProps): ReactNode => {
+  const { retentionPeriodInDays, setRetentionPeriod, isOwner } = props;
+
+  const daysId = useUniqueId('log-retention-days');
+  const descriptionId = useUniqueId('log-retention-description');
+  const formId = useUniqueId('log-retention-form');
+
+  return (
+    <>
+      <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', marginTop: '.75rem' }}>
+        <FormLabel
+          htmlFor={formId}
+          style={{ fontWeight: 600, whiteSpace: 'nowrap', marginRight: '.5rem', marginTop: '.5rem' }}
+        >
+          Workflow Log Retention:
+        </FormLabel>
+      </div>
+      <div id={descriptionId} style={{ marginTop: '.5rem', fontSize: '12px' }}>
+        This{' '}
+        <ExternalLink href='https://cloud.google.com/logging/docs/buckets#custom-retention'>
+          log bucket retention setting
+        </ExternalLink>{' '}
+        specifies the number of days to retain workflow task logs in the bucket. After the retention period has passed,
+        logs will be deleted. Valid values range from 1 to 3650 days (10 years).{' '}
```

Comment:
* I think the fact that there's a bucket involved is an implementation detail that users don't need to know about - I would remove the word "bucket" from this text and just refer to log retention.
 * This setting will also control retention of compute environment VM logs - I'm not sure how often users look at those, though. What do you think about calling it Workspace Log Retention and saying "This log retention setting specifies the number of days to retain workspace logs, including those from workflow tasks and compute environment VMs. After the retention period..."

Reply:
That's a good point. I can revise it.

Reply:
I have updated the description to include both workflow logs and compute environment VM logs in it.

After talking with Beth, she suggested to rename the title to `Workspace Analysis Log Retention` so that its not too vague and lets users know that this pertains to analysis type logs.